tsconfig.json

```diff
@@ -42,7 +42,7 @@
     "noFallthroughCasesInSwitch": true,
     "noUncheckedIndexedAccess": true,
     "inlineSources": true,
-    "useDefineForClassFields": false,
+    "useDefineForClassFields": true,
```

Comment:
This is a behavior change though, right? Previously, it was as if this flag was false?

Reply:
This is because ts's class property is added before ES standard and have conflct. When targeting es2022 a tsconfig option for compatibility useDefineForClassFields is default true instead of false.

No behavior change.

Reply:
I may be not clear enough... set this to true is to keep behavior to not change